src/git-town-rename-branch

```diff
@@ -7,6 +7,32 @@ function preconditions {
   destination_branch_name=$2
   force_parameter=$3
 
+  if (( $# == 0 )); then
+    echo_error_header
+    echo_error "No branch name provided to rename to."
+    exit_with_error newline
+  elif (( $# == 1 )); then
+    target_branch_name=$(get_current_branch_name)
+    destination_branch_name=$1
+  elif (( $# == 2 )); then
+    if [[ $2 == "-f" ]]; then
+      target_branch_name=$(get_current_branch_name)
+      destination_branch_name=$1
+      force_parameter=$2
```

Comment:
Can we get rid of the `force_parameter` variable now in favor of just using the `force_renam` variable?